chaos_genius/views/anomaly_data_view.py

```diff
@@ -390,6 +395,50 @@ def kpi_anomaly_retraining(kpi_id):
     else:
         return jsonify({"msg" : f"retraining failed for KPI: {kpi_id}, KPI id is None"})
 
+@blueprint.route("/<int:kpi_id>/download_anomaly_data", methods=["GET"])
```

Comment:
(flake8) E302 expected 2 blank lines, found 1
```suggestion
        return jsonify({"msg" : f"retraining failed for KPI: {kpi_id}, KPI id is None"})


@blueprint.route("/<int:kpi_id>/download_anomaly_data", methods=["GET"])
```